src/middlewares/checkSuperAdmin.ts

```diff
@@ -0,0 +1,11 @@
+import * as Utils from "../utils/response";
+
+const checkSuperAdmin = async (req: any, _res: any, next: any) => {
+  if (req.user.isAdmin == 2) {
```

Comment:
isAdmin seems like a boolean variable. But you are storing number?

Reply:
Actually , this is kept INT because 0 will represent a normal user, 1 represents admin and 2 represents superAdmin, which will one of us developers